_sources/changelog.txt

```diff
@@ -1,16 +1,6 @@
 Change Log
 ==========
 
-0.11.2
-------
-
-- fix for a second deployment regression
-
-0.11.1
-------
-
-- fix for a deployment regression
-
```

Comment:
yeah, no. I'm not going to deploy docs for a not-yet released version of datacats (because of stuff like this)